README.md

```diff
@@ -92,38 +92,35 @@ sudo ./install-dependencies.sh && CC=clang CXX=clang++ ./build.sh
 
 For quicker dev setup, we provide a [docker image](tools/docker/README.md) with the toolchain installed.
 
-# Beta builds
+# Release candidate builds
 
 For those of us who like to live on the edge!
```

Comment:
I would remove this. It makes it sound like using our software is dangerous and non-native english speakers might be confused about this.